ckanext/validation/templates/validation/validation_read.html

```diff
@@ -1,51 +1,70 @@
-{% extends "package/base.html" %}
+{% extends "package/resource_edit_base.html" %}
 
 
-{%- block subtitle %}{{ _('Validation Report') }}{% endblock -%}
+{%- block subtitle %}{{ _('Validation Report') }}{% endblock subtitle -%}
 
-{% block breadcrumb_content_selected %}{% endblock %}
+{% block breadcrumb_content_selected %}{% endblock breadcrumb_content_selected %}
 {% block breadcrumb_content %}
   {{ super() }}
-
-  <li><a href="{{ h.url_for('resource.read', id=dataset.id, resource_id=resource.id ) }}">{{ h.resource_display_name(resource)|truncate(30) }}</a></li>
-  <li class="active"><a href="">Validation Report</a></li>
-{% endblock %}
+{% endblock breadcrumb_content %}
 
 {% block pre_primary %}
+  {% asset "ontario_theme_scripts/ontario_theme_step_indicator" %}
```

Comment:
You can remove this line from this file and add it into the `ckanext-ontario_theme/ckanext/ontario_theme/templates/internal/base.html` under the `{% block links %}` block